src/validate.ts

```diff
@@ -189,6 +194,9 @@ export function response(compiledPath: CompiledPath | undefined,
   if (response === undefined) {
     response = operation.responses['default'];
   }
-
+  // make Date or ObjectId to string
```

Comment:
It's a little unclear what this is fixing, can you add a test case that was failing before that now works with this PR?  Thanks!